manifests/repositories.pp

```diff
@@ -20,8 +20,8 @@
         }
       } else {
         yumrepo { 'htcondor-stable':
-          descr    => "HTCondor Stable RPM Repository for Redhat Enterprise Linux ${major_release}",
-          baseurl  => "http://research.cs.wisc.edu/htcondor/yum/stable/rhel${major_release}",
+          descr    => "HTCondor Stable RPM Repository for Redhat Enterprise Linux ${facts['os']['release']['major']}",
+          baseurl  => 'http://research.cs.wisc.edu/htcondor/yum/stable/rhel$releasever',
```

Comment:
To be more specific: It looks like the default location for yum to get this value from (`/etc/redhat-release`) now includes an additional patch version and is parsed incorrectly.